bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -294,6 +294,17 @@ impl PartialOrd for LockTime {
     }
 }
 
+impl fmt::Debug for LockTime {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        use LockTime::*;
+
+        match *self {
+            Blocks(ref h) => write!(f, "{} blocks", h),
+            Seconds(ref t) => write!(f, "{} seconds", t),
```

Comment:
"X seconds" sounds like a duration to me and not a time.. Why is this variant named `Seconds` here? Instead of `Timestamp`? I mean for pretty printing, it sucks std doesn't to iso datetime prints or so, because as a timestamp, it would be something like that, but without that I would have printed something like `"unix timestamp: {}"` or something.

Reply:
Hmm, interesting point. It's named "seconds" because this is in units of seconds, much like the "blocks" variant is in units of blocks.

But in the case of blocks, the 0 point is zero so there isn't really a distinction between a duration and a timestamp. For time there is.

I wouldn't mind renaming the variant itself to `Timestamp` or `UnixTimestamp`. Though that would be a breaking change that would need to wait for the next release. And I'm not totally convinced the value is worth the breakage.

Reply:
Concept ack improving the debug output, note in `Display` (alternate) we have
```rust
Seconds(ref t) => write!(f, "block-time {} (seconds since epoch)", t),
```

I had quick look and I believe I went with `Seconds(Time)` to match `Blocks(Height)`. This also was done to fit in with `relative::LockTime` which uses `Time` as the variant name. I'm not claiming in anyway that I named things perfectly but I would warn against changing an identifier without being willing to put in a decent amount of time to going over both modules because all the docs and nomenclature is uniform throughout both modules, well at least I believe it is, I put many hours into those two modules because the time locks are so confusing.